README.md

```diff
@@ -9,19 +12,32 @@ A release is a version of your code that is deployed to an environment. When you
 Additionally, releases are used for applying source maps to minified JavaScript to view original, untransformed source code.
 
 ## Usage
+Adding the following to your workflow will create a new Sentry release for the 
+`production` environment of your project and upload JavaScript source maps from
+the `./lib` directory. 
+  
+```yaml
+uses: getsentry/action-release@v1
+with:
+  environment: 'production'
+  sourcemaps: './lib'
+```
 
+Don’t forget to include the following environment variables:
 ```yaml
-steps:
-...
-- name: Sentry Release
-  uses: getsentry/action-release@v0.0.1
-  with:
-    environment: 'production'
-    sourcemaps: './lib'
+env:
+  SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+  SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+  SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
 ```
 
 ## Troubleshooting
+Building and running this action locally on unsupported environment yields an error that looks like:
```

Comment:
I think this should read either:

Building and running this action locally on an unsupported environment yields an error that looks like:

or 

Building and running this action locally on unsupported environments yields an error that looks like: